src/test/java/teammates/test/cases/action/AdminEmailLogPageActionTest.java

```diff
@@ -81,8 +80,12 @@ private void loadLogMessages() {
     }
 
     private void initVariable() {
-        formatterAdminTime = new SimpleDateFormat("dd/MM/yy");
-        formatterAdminTime.setTimeZone(TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE));
+        formatterAdminTime = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+    }
+
+    private String formatInstantInAdminTimeZone(Instant instant) {
```

Comment:
Rename to `formatDateInAdminTimeZone`, since the output only contains date and not a datetime.

Reply:
`Date` may be understood as the java class `java.util.Date` though. I'll change the name to `formatInstantAsDateInAdminTimeZone`